kork/kork-artifacts/src/main/java/com/netflix/spinnaker/kork/artifacts/artifactstore/filters/ApplicationFilter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 Apple Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.kork.artifacts.artifactstore.filters;
+
+import com.netflix.spinnaker.security.AuthenticatedRequest;
+import java.util.List;
+
+public class ApplicationFilter {
```

Comment:
javadoc please, for the class and method.  Especially please clarify whether filter here means include or exclude.